server/schedulers/balance_region.go

```diff
@@ -189,11 +192,14 @@ func (s *balanceRegionScheduler) Schedule(cluster opt.Cluster) []*operator.Opera
 
 			if op := s.transferPeer(plan); op != nil {
 				op.Counters = append(op.Counters, schedulerCounter.WithLabelValues(s.GetName(), "new-operator"))
-				return []*operator.Operator{op}
+				result = append(result, op)
+				if len(result) >= s.conf.Batch {
```

Comment:
It's better to add a test for creating multiple operators.

Reply:
done